study/src/main/java/cache/com/example/cachecontrol/CacheWebConfig.java

```diff
@@ -1,13 +1,21 @@
 package cache.com.example.cachecontrol;
 
 import org.springframework.context.annotation.Configuration;
+import org.springframework.http.CacheControl;
 import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import org.springframework.web.servlet.mvc.WebContentInterceptor;
 
 @Configuration
 public class CacheWebConfig implements WebMvcConfigurer {
 
     @Override
     public void addInterceptors(final InterceptorRegistry registry) {
+        CacheControl cacheControl = CacheControl.noCache().cachePrivate();
+
+        WebContentInterceptor webContentInterceptor = new WebContentInterceptor();
+        webContentInterceptor.addCacheMapping(cacheControl, "/**");
```

Comment:
```suggestion
        WebContentInterceptor webContentInterceptor = new WebContentInterceptor();
        webContentInterceptor.addCacheMapping(CacheControl.noCache().cachePrivate(), "/**");
```
혹시 `cacheControl` 을 변수로 추출한 이유가 있었을까요?
개인적으론 설정 메서드와 그 메서드에 사용되는 설정값들은 한 곳에 모여 있어야 훨씬 파악이 쉽다고 생각해요. 
제안드린 변경안과 비교해보시고 마음에 드시는 쪽으로 선택하시면 좋을 듯 합니다!

Reply:
변수로 추출한 별다른 이유는 없었습니다. 확실히 모여 있는 것이 파악하기 쉬운 것 같네요!!